app/pages/events/edit/ActivityEditForm.php

```diff
@@ -32,24 +26,26 @@
     $activity = new Activity();
 }
 
+$item_name = ($event && !$event?->type == EventType::Simple) ? "activité" : "événement";
```

Comment:
We could call both of them "event". I think it would be OK for users and make the code simpler